### DISCUSSION TITLE
PyAnsys error: modal participation operator

### DISCUSSION THREAD

**Discussion by mredoutey** [ansys/pydpf-core#1013] [Q&A] [answered]
I am using the PyAnsys package to do a modal analysis and I’m having problems with the modal_participation operator (https://dpf.docs.pyansys.com/version/0.8/api/ansys.dpf.core.operators.math.modal_participation.html). 
 
I am trying to instantiate the operator using: op = dpf.operators.math.modal_participation()
 
I am getting the following Key error: “The operator harmonic::modal_participation doesn’t exist in the registry. Check its spelling in the documentation or verify its availability in your Context (Entry/Premium) and in your loaded plugins.” 
 
Can anyone help me determine what is causing this error?

Thanks!

**Comment by PProfizi** [comment]
Hi @mredoutey,

Could you please provide more information such as the Ansys version you are using (or the DPF server version), as well as a list of the ansys packages installed in your Python environment (using `pip list`)? That would enable me to better target the exact issue.

**Reply by mredoutey** [reply]
Hi @PProfizi,

I am using Python version 3.9.2. My requirements.txt file is attached.
[requirements.txt](https://github.com/ansys/pydpf-core/files/12004090/requirements.txt)

Thank you!

**Reply by PProfizi** [reply]
Hi @mredoutey,
Thank you for this information. What I would still need is the version of your local Ansys installation as I see from your requirements.txt that you do not use the standalone DPF server which would appear as ansys-dpf-server.

**Reply by mredoutey** [reply]
Hi @PProfizi,

I have Ansys 2022 R2 installed locally.

**Reply by PProfizi** [reply, marked as answer]
@mredoutey the modal_participation operator appears in the doc starting with ansys-dpf-core 0.8, which means it will only be available starting with our 2023 R2-pre1 release of the DPF server.
You can download our DPF server and use the same license or license server as usual by following the information [here](https://dpf.docs.pyansys.com/version/stable/user_guide/getting_started_with_dpf_server.html).

The standalone DPF server is our mean to provide clients and developers with DPF functionalities updated more frequently than the usual 6 months release cycle.

As for marking the feature in the doc as requiring a minimal version, this is en enhancement we are actively working towards.
Sorry for the lack of information to that regard.

Please do not hesitate if you have more questions or need help with installing the standalone DPF Server.

**Reply by mredoutey** [reply]
@PProfizi I see, that makes sense. I will download the DPF server and see if that solves the issue. Thank you!

**Reply by mredoutey** [reply]
@PProfizi this worked! Thank you!
